packages/react-input/Spec-variants.md

```diff
@@ -0,0 +1,85 @@
+# Input Variants
```

Comment:
Clarification: this is NOT the main spec and not intended to be 100% finished yet, it's just notes about tentative plans and some comparison with v8 since that's what I'm most familiar with. A detailed spec and full comparison for any of these variants will be filled out once we're closer to implementing them.